{% extends "layout.html" %}
  
{% block title %}Project Description{% endblock %}
{% block lead %}{% endblock %}

{% block page_content %}

<div class = "page-container">
  <aside class="sidebar">
    <ul class="sidebar-nav">
      <li><a href="#abstract" class="active">Abstract</a></li>
      <li><a href="#before">Before Skippit: The world of transcription regulation</a></li>
      <li><a href="#after">After Skippit: Beyond the Limits of DNA</a>
        <ul class = "sidebar-subnav">
          <li><a href="#new">A new design space: Functional RNA Elements (FREs)</a>
          <li><a href="#model">TADPOLE: engineering ligand-responsive FRE switches</a>
            <ul class = "subnav">
              <li><a href="#conceptual">Conceptual overview</a></li>
              <li><a href="#architecture">Molecular architecture and rationale</a></li>
              <li><a href="#on-off">ON and OFF</a></li>
            </ul>
          </li>
          <li><a href="#automated">Automated design</a></li>
        </ul>
      </li>
      <li><a href="#riboswitch">The SKIPPIT Riboswitch</a>
      <ul class = "sidebar-subnav">
          <li><a href="#switch">Stop Codon Readthrough</a></li>
          <li><a href="#scr">SCR riboswitch Design</a></li>
          <li><a href="#states">ON and OFF states</a></li>
        </ul>
      </li>
      <li><a href="#science">Ressonating science</a>
      <li><a href="#potential">The endless potential of RNA</a>
        <ul class = "sidebar-subnav">
          <li><a href="#offer">Join us, beyond the finish line, and discover all that SKIPPIT has to offer</a></li>
        </ul>
      </li>
      <li><a href="#references">References</a></li>
    </ul>

    <div class ="info-card hp-card" id="hp-card1">
        <div class="card">
          <h3 class="card-title">HP Card</h3>
          <p>Look out for these light bulbs throughout our wiki! They contain information boxes to help you understand how Skippit answers synbio challenges through the expertise of world-class researchers and stakeholders.</p>
          <button class="btn btn-outline" onclick = "window.location.href='human-practices'">Learn More</button>
        </div>
      </div>
  </aside>
  
  <div class = "page-content">
    <div class = "header">
      <h1 class = "page-title">Project Description</h1>
      <p class = "page-desc">A next generation tool to control genetic expression during translation</p>
    </div>
    <div class = "content"> 
      <section id = "abstract">
        <h1 class ="section-title">Abstract</h1>
        <p class="section-text">
          As synthetic biology positions itself as a field of the future, the need for new tools that allow better control of protein expression and new options for genetic engineering is clear. SKIPPIT is a platform that harnesses the full power of STOP Codon Readthrough elements, from both characterising them to making them into a tool: a readthrough riboswitch. Developing this switch has only been possible thanks to the development of a cutting edge Software: TADPOLE. This software uses computationally-predicted RNA structures to optimise the design of RNA switches from RNA sequences.   
        </p>
      </section>
      <section id = "before">
        <h1 class ="section-title">Before Skippit: The world of transcription regulation</h1>
        <p class="section-text">
          Synthetic biology has cemented itself as the field of the future, as it aims to solve problems by leveraging the power of nature and living organisms [1]. Synthetic biologists have been reprogramming biological systems for the last 25 years, but as the field evolves, new disciplines arise: biosensing, biomanufacturing, custom genomes, artificial cells, new-gen therapeutics…  [2]. 
          <br><br>
          For the past 50 years [3], strategies to regulate gene expression have revolved around transcription factor switches [4], CRISPR interference [5], RNA interference [6], and chemically inducible promoters [7]. While effective, each has limitations: transcriptional control can be slow due to the extra step of mRNA production and processing, RNAi can suffer from off-target effects and limited dynamic range [8], and inducible promoters often require bulky proteins or chemicals that are hard to deliver or interfere with cell metabolism [9]. 
        </p>
        <div>
          <div class="image-container desc-image">
            <img src = "https://static.igem.wiki/teams/5622/images/description/tf-crispr-1.webp" alt = "TF + CRISPR">
            <!-- <p class = "photo-caption">TF + CRISPR</p> -->
          </div>
          <div class="image-container desc-image">
            <img src = "https://static.igem.wiki/teams/5622/images/description/irna-promoter-1.webp" alt = "iRNA + Promoter">
            <!-- <p class = "photo-caption">iRNA + Promoter</p> -->
          </div>
        </div>
        <p>
          As synthetic biology moves into gene therapy, biomanufacturing, and other high-precision fields, there is growing demand for regulation methods that are faster, work at multiple layers, and deliver cleaner, noise-resistant control. Skippit is our answer to that challenge.<span class = "info-word hp-icon" data-target= "hp-card1">&#128161;</span>
        </p> 
      </section>
      <section id = "after">
        <h1 class ="section-title">After Skippit: Beyond the Limits of DNA</h1>
        <p class="section-text">
          An alternative method to control protein expression is found in translation-level control, which provides rapid responses and direct protein regulation without altering transcription [10]. RNA switches—RNA sequences that often control gene expression by interacting with a specific stimuli, that stimuli often being a ligand molecule—are especially attractive in this context: they are compact, programmable, and highly specific. The theophylline-binding aptamer <span class = "info-word hp-icon" data-target= "hp-card1">&#128161;</span> [11], for example, has been extensively used to regulate downstream elements when linked to open reading frames [12, 13].
        </p>
        <h2 id ="new" class = "section-subtitle">A new design space: Functional RNA Elements (FREs)</h2>
        <p class="section-text">With SKIPPIT, we are taking a leap forward in the world of translation-level control, by coupling <span class = "bold-word">Conformational RNA Elements</span> (CREs) to what we define as <span class = "bold-word">Functional RNA Elements</span> (FREs). These FREs are RNA sequences able to carry out their function purely through their own structure. FREs are capable of folding into shapes that control ribosome behaviour [14], alter reading frames [15], or enable stop codon readthrough [16], amongst other functions.</p>
        <div class = "video" style="position: relative; padding-top: 55.83%;">
          <iframe title="Barcelona-UB: Creation of Riboswitches (2025)" width="80%" height="80%" src="https://video.igem.org/videos/embed/wnAAjLnQPJYmF384yfE5J7?loop=1&amp;autoplay=1" style="border: 0px; position: absolute; inset: 0px;" allow="fullscreen" sandbox="allow-same-origin allow-scripts allow-popups allow-forms"></iframe>
        </div>
        <p class="section-text">
          FREs are the “mechanical executors” of the RNA world, as their function is mechanically constricted. Traditionally, they have been limited by their sequence and have not been controlled in a dynamic fashion.
          <br><br>
          However, <span class = "highlight-word">we are changing that with SKIPPIT</span>
        </p>
        <h2 id ="model" class = "section-subtitle">TADPOLE: engineering ligand-responsive FRE switches</h2>
        <h3 id ="conceptual" class = "section-subtitle2">Conceptual overview</h3>
        <p class="section-text">
          The SKIPPIT principle on RNA switch design is to convert a static Functional RNA Element (FRE) into a stimuli-responsive device by physically coupling it to a Conformational RNA Element (CRE), connecting them via a short linker.
        </p>
        <div class = "video" style="position: relative; padding-top: 55.83%;">
          <iframe title="Barcelona-UB: Loop OFF-ON to ON-OFF switch (2025) [English]" width="80%" height="80%"  src="https://video.igem.org/videos/embed/4idARFGe6uvL7xvsxvQnsX?loop=1&amp;autoplay=1" style="border: 0px; position: absolute; inset: 0px;" allow="fullscreen" sandbox="allow-same-origin allow-scripts allow-popups allow-forms"></iframe>
        </div>
        <h3 id ="architecture" class = "section-subtitle2">Molecular architecture and rationale</h3>
        <p class="section-text">Each construct is organised as three modular elements:</p>
        <ol>
          <li><span class="highlight-word">CRE — the ligand sensor.</span> Key requirements: known bound conformation, well-characterised binding pocket, and compact structure. In our proof of concept riboswitch, we have used the <span class="italic-word">theophylline aptamer.</span></li>
          <li><span class="highlight-word">Linker — a short oligomer</span> (we use 6–10 nt as our primary search window) that mediates possible base-pairing between CRE and FRE; acts as the mechanical hinge controlling coupling vs. decoupling. In our proof of concept riboswitch, we have used a linker generated by our software,  <span class="italic-word">Tadpole.</span></li>
          <li><span class="highlight-word">FRE — the functional RNA</span> motif whose activity depends on adopting a specific secondary structure. In our proof of concept riboswitch, we have used a <span class="italic-word">STOP codon Readthrough (SCR)</span> known as SECIS.</li>
        </ol>
        <div class="accordion">
          <div class="accordion-header">
              <div class="accordion-title">Design principles</div>
              <div class="accordion-icon"></div>
          </div>
          <div class="accordion-content">
              <div class="accordion-content-inner">
                <ul class="point-list">
                  <li>The linker must permit specific aptamer &#8596; FRE interactions in the OFF state but avoid creating long, stable helices that trap the molecule kinetically.</li>
                  <li>The aptamer's <span class="bold-word">entry and binding nucleotides</span> must remain accessible for ligand binding in the ON state; in practice, we avoid permanent pairing of the binding pocket bases in the OFF design.</li>
                  <li>Critical base pairs that define FRE function must be intentionally disrupted in the OFF state and recoverable in the ON state.</li>
                </ul>
              </div>
          </div>
        </div>
        <h3 id ="on-off" class = "section-subtitle2">ON and OFF</h3>
        <p class="section-text">
          RNA switches are characterised by having an ON and an OFF state, and this is the main challenge to design new switches: to make these states robust and mutually exclusive, and to induce change in the presence of the external stimuli. 
          <br><br>
          We can differentiate two types of Switches according to their ON and OFF states: <span class="italic-word">ON-ON and ON-OFF</span>:
        </p>
        <div class="desc-imgs">
          <div class="image-container desc-image">
            <img src = "https://static.igem.wiki/teams/5622/images/description/on-on-off-off.webp" alt = "Team photo">
            <p class = "desc-photo-caption">In the ON-ON state, in presence of an external stimuli that interacts with the CRE, the FRE is active. In absence of the stimuli, the FRE is inactive.</p>
          </div>
          <div class="image-container desc-image">
            <img src = "https://static.igem.wiki/teams/5622/images/description/off-on-on-off.webp" alt = "Team photo">
            <p class = "desc-photo-caption">In the ON-OFF state, in presence of an external stimuli that interacts with the CRE, the FRE is inactive. In absence of the stimuli, the FRE is active.</p>
          </div>
        </div>
        <h2 id ="automated" class = "section-subtitle">Automated design</h2>
        <p class="section-text">
          Testing and evaluating all possible RNA switch configurations by hand is a very arduous, extensive task. That’s why we developed <span class="italic-word">TADPOLE</span>, a software that completely automates the design process of these riboswitches: 
          <br><br>
          Because standard Minimum Free Energy (MFE) MFE folding programs cannot fully simulate ligand binding (when using aptamers), miniRNA binding (for theoholds), and so on, we need an alternative to simulate the external signal that changes the conformation of the CRE. Therefore, we  use <span class="bold-word">constrained folding</span> to approximate the effect of the external signal (change of configuration of the CRE). The resulting workflow is:
        </p>
        <div class="accordion">
          <div class="accordion-header">
              <div class="accordion-title">Search space generation</div>
              <div class="accordion-icon"></div>
          </div>
          <div class="accordion-content">
              <div class="accordion-content-inner">
                <ul class="point-list">
                  <li>Input: CRE sequence, FRE sequence.</li>
                  <li>Generate systemlinker candidates across the chosen length range (6–10 nt by default) and with a small set of composition heuristics. (avoid extreme GC runs; prefer mixed composition to reduce off-target pairing).</li>
                </ul>
              </div>
          </div>
        </div>
        <div class="accordion">
          <div class="accordion-header">
              <div class="accordion-title">OFF state prediction</div>
              <div class="accordion-icon"></div>
          </div>
          <div class="accordion-content">
              <div class="accordion-content-inner">
                <ul class="point-list">
                  <li>Run RNA secondary-structure prediction (RNAfold / ViennaRNA) on the full sequence to get the default MFE fold.</li>
                  <li>Analyse pairing patterns: check whether the FRE is disrupted and determine the feasibility of the proposal. inspect the accessibility of FRE functional stems.</li>
                </ul>
              </div>
          </div>
        </div>
        <div class="accordion">
          <div class="accordion-header">
              <div class="accordion-title">ON state approximation</div>
              <div class="accordion-icon"></div>
          </div>
          <div class="accordion-content">
              <div class="accordion-content-inner">
                <ul class="point-list">
                  <li>Impose light constraints to mimic the effects of the external signal on the CRE  (change on conformation on the CRE).</li>
                  <li>Recompute folding; evaluate whether the FRE recovers its required functional stems and whether the aptamer binding pocket remains available.</li>
                </ul>
              </div>
          </div>
        </div>
        <div class="image-container">
          <img src = "https://static.igem.wiki/teams/5622/images/description/updated/tadpole-workflow.webp" alt = "Tadpole">
          <!-- <p class = "photo-caption">OFF-state</p> -->
        </div>
        <p class="section-text">
          TADPOLE automates these steps: it enumerates linkers, runs folding predictions, applies constraints, computes metrics, organises candidates, and produces visualisations (structure plots, pairing maps) and a shortlist ready for synthesis.
          <br><br>
          Our TADPOLE website has been designed to be accessible for anyone: It has a visual interface, users don’t need to know any coding, and the python packages of the software have been published to facilitate the programming of new projects from our code.<span class = "info-word hp-icon" data-target= "hp-card">&#128161;</span>
          <br><br>
          We are opening RNA switch design to any research team, regardless of their expertise.
        </p>
        <div class="desc-imgs">
          <div class="image-container desc-image">
            <img src = "https://static.igem.wiki/teams/5622/images/description/updated/scr-riboswitch-off-on.webp" alt = "Team photo">
          </div>
          <div class="image-container desc-image">
            <img src = "https://static.igem.wiki/teams/5622/images/description/updated/scr-riboswitch-on-off.webp" alt = "Team photo">
          </div>
          <p class = "photo-caption">Images: Depiction of a dual-protein system regulated by our SCR riboswitch. The aptamer is in dark red, whilst the SCR element is in pink</p>
        </div>
        <p class="section-text">It is a specially interesting feature, especially in the world of dual-protein control, as in one single transcript our SKIPPIT SCR switch can simultaneously regulate the translation of two proteins.</p>
      </section>
      <section id="riboswitch">
        <h1 class="section-title">The SKIPPIT Riboswitch</h1>
        <p class="section-text">The tools we developed in the Dry Lab allowed us to design a brand new riboswitch with a completely novel function: a Theophylline-responsive STOP codon readthrough switch.</p>
        <h2 id ="codon" class = "section-subtitle">Stop Codon Readthrough</h2>
        <p class="section-text">STOP codon readthrough elements (SCR) are sequences in mRNA transcripts that interact with the ribosome as it reads the STOP codon. This interaction allows the STOP to be recoded, meaning that instead of termination occurring by the release factor complex, the STOP is decoded by a near-cognate tRNA. This near cognate tRNA recognizes the STOP codon and adds a modified aminoacid, such as selenocysteine, to the peptide chain, which allows translation to continue beyond the STOP. [17]</p>
        <div class="video home-video" style="position: relative; padding-top: 56.25%;">
          <iframe title="Barcelona-UB: SCR Translational influence (2025)" width="100%" height="100%" src="https://video.igem.org/videos/embed/9U3qG3tzMAeG285RcXMthZ?loop=1&amp;autoplay=1" style="border: 0px; position: absolute; inset: 0px;" allow="fullscreen" sandbox="allow-same-origin allow-scripts allow-popups allow-forms"></iframe>
        </div>
        <div class="accordion">
          <div class="accordion-header">
              <div class="accordion-title">Our project’s SCRs</div>
              <div class="accordion-icon"></div>
          </div>
          <div class="accordion-content">
              <div class="accordion-content-inner">
                <p class="section-text">
                  Throughout our project we have worked with two different SCR elements: SCR-D and SECIS. SCR-D is a new SCR element not yet available to the public that has been barely studied. During our project, we set out to better characterise it in an attempt to better understand its inner workings. As a result, we were able to generate three new SCR elements with variable readthrough rates. 
                  <br><br>
                  On the other hand, SECIS is much better characterised part, publicly available and whose function heavily relies on its structure. Due to these factors, we decided to use SECIS as the SCR element to generate our SKIPPIT SCR riboswitch.
                </p>
              </div>
          </div>
        </div>
        <h2 id ="scr" class = "section-subtitle">SCR riboswitch Design</h2>
        <p class="section-text">As the SKIPPIT SCR riboswitch was designed using TADPOLE and its integrated Model, we needed three key parts:
          <ol>
            <li><span class="highlight-word">FRE:</span> We chose a well-characterised STOP codon readthrough (SCR) element known as SECIS (Selenocysteine Insertion Sequence).<span class = "info-word hp-icon" data-target= "parts-card1">&#129513;</span></li>
            <li><span class="highlight-word">CRE:</span> We chose a Theophylline aptamer that’s been extensively studied and has been previously used to create riboswitches. <span span class = "info-word hp-icon" data-target= "parts-card2">&#129513;</span></li>
            <li><span class="highlight-word">Linker:</span> With the FRE and CRE sequence input, TADPOLE calculated and generated linkers for our riboswitch. <span span class = "info-word hp-icon" data-target= "parts-card3">&#129513;</span></li>
          </ol>
        </p>
        <div class="image-container">
          <img src = "https://static.igem.wiki/teams/5622/images/description/on-off.webp" alt = "ON-OFF">
          <p class = "photo-caption">Image: Diagram depicting the structural prediction of our SCR switch.</p>
        </div>
        <h2 id ="states" class = "section-subtitle">ON and OFF states</h2>
        <p class="section-text">Our SKIPPIT SCR riboswitches are RNA switches of the ON-OFF type.</p>
        <ul class="point-list">
          <li>OFF-ON state: When the aptamer is unbound, the interactions between the aptamer (CRE element) and SECIS (FRE element) are destabilised, meaning the FRE is free to fold into its active conformation.</li>
          <li>ON-OFF state: When the aptamer is bound to its ligand, the intentional base-pairing interactions between the aptamer and SECIS are established, leading to the disruption of SECIS’ functional fold and its inactive conformation.</li>
        </ul>
        <p class="section-text">Therefore our SKIPPIT riboswitch enables the ribosome to skip the STOP codon in absence of the ligand, allowing translation to continue beyond the STOP. When adding the ligand this dynamic is inverted, and translation ends at the STOP codon.</p>
      </section>
      <section id="science">
        <h1 class="section-title">Ressonating science</h1>
        <p class="section-text">
          Alongside our technical innovation, we have a deep-rooted duty to transmit the inner workings of science to the population in order to truly advance the field and further scientific progress. Without the proper scientific context, unchecked claims by non-experts can come across as truth to the general population, whilst expert fact-based statements can be <span class="italic-word">perceived</span> as deceitful. The inversion between what is true and what is perceived as true can have severe consequences on society. In a world  where AI-fabricated “facts” are on the rise, we have focused on developing an educational toolkit designed to demystify the scientific process and the people behind it, fostering critical thinking and trust in science.
          <br><br>
          Traditional education often falls short in demonstrating the scientific process and the inner workings of breakthrough technologies. To shed light on these, we have developed five adaptable, low-cost activities that present science as a collective, rigorous, creative, and ethical endeavour. Covering ethics, creativity, experimentation, technology, and communication, these activities target secondary education and early college students, and can be tailored by teachers according to their needs. Supported by collaborations with philanthropic foundations, student associations and local governments, our toolkit aims to outlast iGEM and help build a scientifically-literate society.
        </p>
        <div class="image-container">
          <img src = "https://static.igem.wiki/teams/5622/images/description/reineta.webp" alt = "Tadpole">
          <!-- <p class = "photo-caption">OFF-state</p> -->
        </div>
        <p class="section-text">These foundations are essential to ensure that future innovations, like our RNA switches, are understood, responsibly applied, and resilient against pseudoscience and denialism.</p>
      </section>
      <section id="skippit">
        <h1 class="section-title">Behind the SKIPPIT brand</h1>
        <div class="side-img">
          <p class="section-text side"></p>Frogs are remarkably versatile animals that can be found all over the planet. There are native frogs to every inhabitable continent, and they can thrive in a wide variety of biomes: from tropical forests, to tundras and deserts. Our mascot, Ribbi (named after the ribbit sound frogs make) is meant to reflect the versatility of RNA switches to any situation, just as frogs have adapted to any environment.
          <img class="side" src="https://static.igem.wiki/teams/5622/images/description/updated/red-ribbi.webp">
        </div>
        <div class="accordion">
          <div class="accordion-header">
              <div class="accordion-title">Learn more</div>
              <div class="accordion-icon"></div>
          </div>
          <div class="accordion-content">
              <div class="accordion-content-inner">
                <p class="section-text">
                  Ribbi also symbolises a key part of translation, a ribosome (that helped us settle on the name Ribbi-some). Ribbi was inspired after a Red rain frog, one of the few “red” frogs species in the world characterised for being small and round, and its shape reminded us of a ribosome. Rain frogs are special because they are primarily terrestrial, they are expert burrowers but they cannot swim nor jump.
                  <br><br>
                  Ribbi-some represents the ribosome that cannot “jump” over the STOP codon, and therefore needs an extra boost to skip it. We found it befitting to name our project “SKIPPIT”, as a play on the word “ribbit” while also representing the riboswitch we designed to skip stop codons. SKIPPIT’s meaning soon grew thanks to the incorporation of TADPOLE to our project.
                  <br><br>
                  Frogs are well known for undergoing metamorphosis, and they grow from tadpoles into frogs. Our software TADPOLE allows the design of brand new RNA switches, and as such its name should reflect the creation of new tools, just like tadpoles are the creation of new frogs. 
                </p>
              </div>
          </div>
        </div>
      </section>
      <section id="potential">
        <h1 class="section-title">The endless potential of RNA</h1>
        <p class="section-text">
          SKIPPIT is a promising advancement for the future of synthetic biology. Whilst still in its early stages, our work lays the groundwork for the creation of custom RNA-based switches that can be applied to any field.
          <br><br>
          Our STOP codon readthrough riboswitch represents an innovative approach to gene expression regulation by enabling ligand-dependent stop codon readthrough (SCR) between two genes encoded on a single transcript. This method provides precise, reversible, and dynamic control over translation elongation. This system achieves a tunable readthrough rate offering a first-in-class dual-protein regulatory platform, unexplored in traditional approaches.
          <br><br>
          The <a class="link-text" href="{{ url_for('pages', page='members') }}">application</a> possibilities for our riboswitch are endless. It could be used to regulate genes affected by premature nonsense mutations, offering a new technology to study and develop therapies for many diseases such as cancers, Duchenne Muscular Dystrophy and Cystic Fibrosis. It could be a platform to study new genes, and even to build systems to simulate prion diseases. It could be used as a biosensor, and for metabolic engineering in agriculture, bioremediation and even industry. 
          <br><br>
          Together, SKIPPIT’s switch, Model and Software represent a foundational advance that opens up structural RNA regulation as an accessible, programmable tool for synthetic biology and future biotechnological applications.
        </p>
        <h2 id ="offer" class = "section-subtitle">Join us, beyond the finish line, and discover all that SKIPPIT has to offer</h2>
        <ul class="point-list">
          <li><a class="link-text" href="{{ url_for('pages', page='description') }}">Dry Lab</a>: Development of our Model and Tadpole</li>
          <li><a class="link-text" href="{{ url_for('pages', page='description') }}">Parts</a>: Engineered innovation</li>
          <li><a class="link-text" href="{{ url_for('pages', page='description') }}">Wet Lab</a>: Testing the SKIPPIT riboswitch & characterisation of a new SCR component</li>
          <li><a class="link-text" href="{{ url_for('pages', page='education') }}">Education</a>: The many sides of science.</li>
          <li><a class="link-text" href="{{ url_for('pages', page='human-practices') }}">Human Practices</a>: Understanding the need for RNA switches.</li>
          <li><a class="link-text" href="{{ url_for('pages', page='entrepreneurship') }}">Entrepreneurship</a>: The impact of the RNA world.</li>
          <li><a class="link-text" href="{{ url_for('pages', page='education') }}">Education</a>: The many faces of science.</li>
          <li><a class="link-text" href="{{ url_for('pages', page='team') }}">Team</a>: Meet the people who are pushing the boundaries of RNA.</li>
        </ul>
      </section>
      <section class ="references" id="references">
        <h1 class="section-title">References</h1>
        <ol>
          <li>Benner, S. A. & Sismour, A. M. Synthetic biology. Nat Rev Genet 6, 533–543 (2005).</li>
          <li>Gallup, O., Ming, H. & Ellis, T. Ten future challenges for synthetic biology. Engineering Biology 5, 51–59 (2021).</li>
          <li>Five decades of eukaryotic transcription. Nat Struct Mol Biol 26, 757–757 (2019).</li>
          <li>Travers, A. Transcriptional switches: the role of mass action. Physics of Life Reviews 1, 57–69 (2004).</li>
          <li>Bendixen, L., Jensen, T. I. & Bak, R. O. CRISPR-Cas-mediated transcriptional modulation: The therapeutic promises of CRISPRa and CRISPRi. Molecular Therapy 31, 1920–1937 (2023).</li>
          <li>Feng, X. & Guang, S. Functions and applications of RNA interference and small regulatory RNAs. Acta Biochim Biophys Sin (Shanghai) 57, 119–130 (2024).</li>
          <li>Gatz, C. & Lenk, I. Promoters that respond to chemical inducers. Trends in Plant Science 3, 352–358 (1998).</li>
          <li>Jackson, A. L. et al. Expression profiling reveals off-target gene regulation by RNAi. Nat Biotechnol 21, 635–637 (2003).</li>
          <li>Chen, Y. et al. Tuning the dynamic range of bacterial promoters regulated by ligand-inducible transcription factors. Nat Commun 9, 64 (2018).</li>
          <li>Molecular mechanisms of translational control | Nature Reviews Molecular Cell Biology. <a class="link-text" href="https://www.nature.com/articles/nrm1488">Website</a></li>
          <li>Wrist, A., Sun, W. & Summers, R. M. The Theophylline Aptamer: 25 Years as an Important Tool in Cellular Engineering Research. ACS Synth. Biol. 9, 682–697 (2020).</li>
          <li>Suess, B., Fink, B., Berens, C., Stentz, R. & Hillen, W. A theophylline responsive riboswitch based on helix slipping controls gene expression in vivo. Nucleic Acids Res 32, 1610–1614 (2004).</li>
          <li>Therapeutic Applications of Aptamer-Based Riboswitches | Nucleic Acid Therapeutics. <a class="link-text" href="https://www.liebertpub.com/doi/abs/10.1089/nat.2015.0570">Website</a></li>
          <li>Huang, Z., Du, Y., Wen, J., Lu, B. & Zhao, Y. snoRNAs: functions and mechanisms in biological processes, and roles in tumor pathophysiology. Cell Death Discov. 8, 259 (2022).</li>
          <li>An RNA pseudoknot and an optimal heptameric shift site are required for highly efficient ribosomal frameshifting on a retroviral messenger RNA. | PNAS. <a class="link-text" href="https://www.liebertpub.com/doi/abs/10.1089/nat.2015.0570">Website</a></li>
          <li>Martitz, J. et al. Factors impacting the aminoglycoside-induced UGA stop codon readthrough in selenoprotein translation. Journal of Trace Elements in Medicine and Biology 37, 104–110 (2016).</li>
          <li>Smoljanow, D., Lebeda, D., Hofhuis, J. & Thoms, S. Defining the high-translational readthrough stop codon context. PLoS Genet 21, e1011753 (2025).</li>
        </ol>
      </section>
    </div>
  </div>
</div>

<div class="back-to-top" id="back-to-top">
  ↑
</div>

<style>
.column {
  float: left;
  width: 50%;
  padding: 5px;
}
.row::after {
  content: "";
  clear: both;
  display: table;
}

.desc-image{
  width:50%;
  float:left;
  padding:20px;
}

.desc-photo-caption{
  font-style: normal;
  color:var(--text);
  padding-bottom:1rem;
}
.side-img {
  display: flex;
  align-items: center;
  gap: 1rem; 
}

.side-img p {
  flex: 1;
}

.side-img img {
  width: 100%;
  height: auto;
  margin-bottom:2rem;
}
</style>
{% endblock %}
